src/generate/index.js

```diff
@@ -12,14 +13,15 @@ const normalizeName = require('./overrides')
 // Btree size
 const CHILDREN = 32
 
-// All data is stored in an ipfs folder called data
-// this is the hash of that folder. It includes three files
+// All data is stored in an ipfs folder called DATA_HASH
+// It includes two files
 //
-//     data
-//     |- blocks.csv
-//     |- countries.csv
-//     |- locations.csv
-const DATA_HASH = 'QmTMh5Q1CnB9jV774aKCvPSqibwDy9sJmo7BCThD5f1oY3'
+//     DATA_HASH
+//     |- locationsCsv
+//     |- blocksCsv
+const DATA_HASH = 'bafybeid3munsqqt36qhoumn3kvgwmft6dsswzgl3wiohsanlyqemczcsvi' // GeoLite2-City-CSV_20201013
+const locationsCsv = 'GeoLite2-City-Locations-en.csv'
```

Comment:
❓ Are we going to use other languages in the near future? I see that we have available `de, en, es, fr, ja, pt-BR, ru and zh-CN`

Reply:
Yes, this is certainly a possibility now, but not in the scope of this PR.
Created https://github.com/ipfs-shipyard/ipfs-geoip/issues/81 to track that.